setup.py

```diff
@@ -30,9 +30,6 @@
     'Programming Language :: Python :: 3.10',
     'Programming Language :: Python :: 3.11',
     'Framework :: Django',
-    'Framework :: Django :: 2.2',
-    'Framework :: Django :: 3.0',
-    'Framework :: Django :: 3.1',
     'Framework :: Django :: 3.2',
     'Framework :: Django :: 4.0',
     'Framework :: Django :: 4.1',
```

Comment:
Do we want to add 5 now?

Reply:
I reverted the 5.0 addition, as we can wait for it until it turns stable to be added in the classifiers.